Config/config.php

```diff
@@ -114,6 +122,29 @@
                 'class' => 'MauticPlugin\MauticMediaBundle\Integration\MediaIntegration',
             ],
         ],
+
+        'other' => [
+            'plugin.media.report.cache' => [
+                'class'     => 'Doctrine\Common\Cache\FilesystemCache',
+                'arguments' => [
+                    '"'.__DIR__.'/sql"',
```

Comment:
You will not have write access to the folder of the plugin in production, only the cache folder, see other uses of FilesystemCache. Probably best to not have this as an argument because this can be cached, while the base directory may shift during the deployment by symlinks.

Reply:
Is it alright if I replace it with the kernel.cache_dir parameter? 
 
'%kernel.cache_dir%', works, but I can't seem to be able to append a constant string to the end of it, do you know of a way that works? 

I've tried: 
- '%kernel.cache_dir%"/sql"',
- '%kernel.cache_dir%' .  '"/sql"',
None of those worked...

Looking at: https://developer.mautic.org/#services  "parameters"

Reply:
$sql_query_cache = $container->getParameter('kernel.cache_dir') . "/sql";
Thanks @westonwatson,  @scottshipman, and @heathdutton 
🔥 💯 👍